funnel/views/commentvote.py

```diff
@@ -99,42 +100,41 @@ def new_comment(self):
                     parent = Comment.query.filter_by(
                         suuid=commentform.parent_id.data
                     ).first_or_404()
-                    if (
-                        parent.user.email
-                    ):  # FIXME: https://github.com/hasgeek/funnel/pull/324#discussion_r241270403
-                        if (
-                            parent.user == self.obj.user
-                        ):  # check if parent comment & proposal owner are same
-                            if (
-                                not current_auth.user == parent.user
-                            ):  # check if parent comment is by proposal owner
+                    if parent.user.email:
+                        # FIXME: https://github.com/hasgeek/funnel/pull/324#discussion_r241270403
+                        if parent.user == self.obj.owner:
+                            # parent comment is by the proposal owner
+                            if not parent.user == current_auth.user:
+                                # parent comment is not by the curernt user
                                 send_mail_info.append(
                                     {
-                                        'to': self.obj.user.email or self.obj.email,
-                                        'subject': u"{project} Funnel: {proposal}".format(
+                                        'to': self.obj.owner.email or self.obj.email,
+                                        'subject': u"[New Comment] {project}: {proposal}".format(
```

Comment:
Sentence case: `[New comment]`

Reply:
Using `[💬]`. Let's try it for sometime. I'll remove the `[` `]` if they dont look good. Only `💬` before the subject wasn't looking good to me.